locale/en.po

```diff
@@ -1765,13 +1765,9 @@ msgstr ""
 
 msgid "EDITOR_TUTORIAL_EDITOR_TEXT"
 msgstr ""
-"Welcome to the [b]Microbe Editor[/b].\n"
+"This early in the history of life, the biosphere is pretty simple. Later, the menus you see here will be essential to understanding the interconnected story unfolding on your alien world.\n"
```

Comment:
I feel slightly bad about removing the part that tells the player that they are in the microbe editor as that's a pretty core term that we developers would have then a hard time talking to newer fans of the game if they never see the term anywhere.